pkgs/applications/office/espanso/default.nix

```diff
@@ -3,67 +3,116 @@
 , rustPlatform
 , pkg-config
 , extra-cmake-modules
+, dbus
 , libX11
 , libXi
 , libXtst
 , libnotify
+, libxkbcommon
 , openssl
 , xclip
 , xdotool
+, setxkbmap
+, wl-clipboard
+, wxGTK32
 , makeWrapper
 , stdenv
 , AppKit
 , Cocoa
 , Foundation
+, IOKit
+, Kernel
+, waylandSupport ? false
+, testers
+, espanso
 }:
-
+let
+  x11Support = !waylandSupport && stdenv.isLinux;
+in
```

Comment:
```suggestion

```
we want to expose that

Reply:
This is what I had changed it to earlier, but I reverted to using solely waylandSupport because x11Support && waylandSupport is not supported for espanso.

I can split it in 2 options, but then I think an assertion like `stdenv.isLinux -> (!waylandSupport || !x11Support)` is needed.

Reply:
It is better to use the assertion in this case, something like `assert (not (x11Support && waylandSupport));`

It can be useful in a hypothetical future when we group "global flags" like Sourcemage.

Reply:
I've added a few assertions. I've opted for `waylandSupport != x11Support` to make sure `!(!waylandSupport && !x11Support)` is also covered.